fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryInUnionPlan.java

```diff
@@ -262,6 +278,11 @@ public String getBindingName() {
 
         @Nonnull
         protected abstract List<Object> getValues(@Nonnull EvaluationContext context);
+
+        @Override
+        public int planHash(@Nonnull final PlanHashKind hashKind) {
+            return BASE_HASH_FOR_SOURCES.planHash(hashKind);
```

Comment:
Is this meant to include `bindingName`? It has been removed from the child classes.

Reply:
No -- binding name is not included (or not anymore). Internal names are meaningless as long as the correlated subqueries correctly refer to them. They are even unstable as they are randomly created as opposed to in a central location like the `InExtractor`. I know that this weakens some of the assumptions we make in other parts.